### DISCUSSION TITLE
any plans to support SCTP multi-homing support DSR mode ?

### DISCUSSION THREAD

**Discussion by megarajan** [loxilb-io/loxilb#901] [General]
I am aware that docs say that SCTP multi-homing is supported only in full NAT mode but are there any plans to support SCTP multi-homing support in DSR mode ?

**Comment by TrekkieCoder** [comment]
Many of loxilb's requirement are driven by the community. If there is a strong use-case to support it and subsequent plans for deployment of the same, loxilb developers will surely consider it in the roadmap.

**Comment by megarajan** [comment]
ok thanks for the response... will keep you posted ...
